config/environments/production.rb

```diff
@@ -79,4 +79,8 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.survivor_club = {
+      websocket_url: 'survivor-club.herokuapp.com/websocket'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.